builds/fabric8-che/pom.xml

```diff
@@ -265,4 +265,22 @@
             <url>https://maven.codenvycorp.com/content/repositories/codenvy-public-snapshots/</url>
         </pluginRepository>
     </pluginRepositories>
+    <build>
```

Comment:
Why do we need that ? isn't the license-maven-plugin inherited from upstream project ?

Also you should remove this part: https://github.com/redhat-developer/rh-che/blob/master/builds/fabric8-che/plugins/che-plugin-bayesian-lang-server/pom.xml#L84-L92

Reply:
You were right, Its inherited from the upstream project. I also removed that part from the Bayesian language server pom.xml, I'm updating now

Reply:
great :+1: